addons/binding/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/internal/ChromecastCommander.java

```diff
@@ -52,7 +55,7 @@ public void handleCommand(final ChannelUID channelUID, final Command command) {
         }
 
         if (command instanceof RefreshType) {
-            handleRefresh();
```

Comment:
Is this method still used?

Reply:
Yes it is. It was one the main causes of all the troubles. If you have all the ~30 channels linked it would unconditionally start refreshing the Chromecast state for each channel being linked. Since these refreshes take a while, the refreshes could still be busy while the handler was already disposed causing the warnings shown in the community thread.

There was also a path where it would restart the periodic refresh while the handler was already disposed. :disappointed: 

The refresh is now asynchronous using at most one refresh job. Since it's the same as the periodic refresh job, it is also cancelled when the handler is disposed. :smiley: